packages/frontend/components/composed/article/ArticleText/ArticleText.tsx

```diff
@@ -6,7 +6,14 @@ import * as Styled from "./ArticleText.styles";
 import { ContentImage, FullText } from "components/atomic";
 import { BackToTopBlock } from "components/layout";
 import { ArticleTextFragment$key } from "@/relay/ArticleTextFragment.graphql";
-import AssetInlinePDF from "components/composed/asset/AssetInlinePDF";
+// import AssetInlinePDF from "components/composed/asset/AssetInlinePDF";
+
+import dynamic from "next/dynamic";
+
+const AssetInlinePDF = dynamic(
+  () => import("components/composed/asset/AssetInlinePDF"),
+  { ssr: false }
+);
```

Comment:
@1aurend - After fussing with this and doing some digging, this code was the missing piece. I also added a isMounted check before rendering the react-pdf component. https://github.com/wojtekmaj/react-pdf/issues/136#issuecomment-716643894

The rest was just me playing around with prev/next buttons to make sure it worked 😅

Reply:
Thanks for figuring this out and sharing!